### ISSUE TITLE
Broken Updater Styling in Fullscreen Mode

### ISSUE THREAD

**Issue by Bobrokus** [hawkeye116477/playnite-legendary-plugin#114]
### Prerequisites

- [X] I've looked through [existing issues](https://github.com/hawkeye116477/playnite-legendary-plugin/issues) to make sure that this bug hasn't been reported before.
- [X] I've read the [GitHub Community Guidelines](https://docs.github.com/en/github/site-policy/github-community-guidelines) and know netiquette.

### Playnite version

10.34

### Plugin version

60ee058dc261f5ba2f8e96de7f319ea565b2bbdd

### Legendary Launcher version

0.20.34

### Bug description

The styling of the updater window is broken.

Note: The fullscreen styling is probably not done yet, in that case, this issue serves as a reminder that this problem exists

### Relevant log

none

### Screenshots

![image](https://github.com/user-attachments/assets/401e5f67-cca8-4bb2-bccc-1e69354bd2ca)

**Comment by hawkeye116477** [comment]
> Note: The fullscreen styling is probably not done yet, in that case, this issue serves as a reminder that this problem exists

Yeah, in case of styling I'm not sure if something can be done at P10, perhaps will need to wait until P11.

**Comment by hawkeye116477** [comment]
And here's final result
